packages/cra-template/template.json

```diff
@@ -52,7 +52,7 @@
       "cypress:open": "cypress open",
       "eject": "react-scripts eject",
       "check-circular-modules": "ackee-react-scripts check-circular-modules",
-      "localize": "ackee-localize-spreadsheet-sdk",
+      "localize": "lokse update",
       "postinstall": "yarn localize",
```

Comment:
- I'd remove the `postinstall` hook and let developers to do it manually as it's on FS and FN projects.
- And I'd add `"localizations": "lokse open",`. We have that on FN and it's very useful.

Reply:
✅